packages/engine/Specs/DataSources/EntityViewSpec.js

```diff
@@ -125,6 +156,13 @@ describe(
       const view = new EntityView(entity, scene);
       view.update(JulianDate.now());
     });
+
+    it("update returns with entity.trackingReferenceFrame set to INERTIAL and without entity.orientation property.", function () {
+      const entity = new Entity();
+      entity.trackingReferenceFrame = TrackingReferenceFrame.INERTIAL;
+      const view = new EntityView(entity, scene);
+      view.update(JulianDate.now());
```

Comment:
Should there be an `expect` assertion in this test?

Reply:
In fact this test is no more needed as the case descibed here has the AUTODETECT fallback...